src/main/grammars/RustParser.bnf

```diff
@@ -523,17 +523,15 @@ EnumVariant ::= OuterAttr* identifier VariantArgs? {
   elementTypeFactory = "org.rust.lang.core.stubs.StubImplementationsKt.factory"
 }
 
-private VariantArgs ::= VariantBlockFields | VariantTupleFields | VariantDiscriminant
+private VariantArgs ::= BlockFields | TupleFields | VariantDiscriminant
 
-BlockFields ::= <<mk_block_fileds FieldDecl>> {
+BlockFields ::= '{' <<field_with_recover FieldDecl>>* '}' {
```

Comment:
This should work

```
BlockFields ::= '{' FieldDecl_with_recover* '}'

 private FieldDecl_with_recover ::= !'}' FieldDecl (',' | &'}') {
    pin = 1
    recoverWhile = Field_recover
  }
  private Field_recover ::= !('}' | '#' | pub | identifier)
  ```

We can't remove this rule entirely, but we can get rid of meta `<<` :)